README.md

```diff
@@ -1,9 +1,9 @@
-# Follow JS best practice
+# Leaderboard : Setup
 <a name="readme-top"></a>
 
```

Comment:
- [ ] [OPTIONAL]
I think the ReadMe has less spacing which makes it look cluttered. Can we improve upon this by ensuring consistent spacing as seen in this ReadMe [template](https://github.com/microverseinc/readme-template)?